### PR TITLE
[dotnet] IAST SqlInjection

### PR DESCRIPTION
**Pull request by daniel-romano-DD**
## Description
IAST SqlInjection for dotnet

## Motivation
IAST system tests for dotnet platform

## Workflow

1. ⚠️⚠️ Create your PR as draft
2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

Once your PR is reviewed, you can merge it! :heart:

## Reviewer checklist

* [ ] Check what scenarios are modified. If needed, add the relevant label (`run-parametric-scenario`, `run-profiling-scenario`...). If this PR modifies any system-tests internal, then add the `run-all-scenarios` label ([more info](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md)). 
* [ ] CI is green
   * [ ] If not, failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] if any of `build-some-image` label is present
  1. is the image labl have been updated ? 
  2. just before merging, locally build and push the image to hub.docker.com
* [ ] if a scenario is added (or removed), add (or remove) it in system-test-dasboard nightly

### CODE REVIEW

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
Some Lint checks failures

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
Why are test failures on dotNet scenarios? are related with this PR? I can't see these errors on the main branch

File: tests/test_semantic_conventions.py [hunk 1]
```diff
@@ -228,6 +228,7 @@ def validator(span):
     @bug(library="cpp", reason="language tag not implemented")
     @bug(library="php", reason="language tag not implemented")
     @bug(library="java", reason="language tag implemented but not for all spans")
+    @bug(library="dotnet", reason="language tag implemented but not for all spans. https://datadoghq.atlassian.net/browse/AIT-8735")
```
Comment (robertomonteromiguel) [inline tests/test_semantic_conventions.py]: If you add only reason="AIT-8735" it would be fine

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]

File: tests/test_semantic_conventions.py [hunk 2]
```diff
@@ -228,6 +228,10 @@ def validator(span):
     @bug(library="cpp", reason="language tag not implemented")
     @bug(library="php", reason="language tag not implemented")
     @bug(library="java", reason="language tag implemented but not for all spans")
+    @bug(
+        library="dotnet",
+        reason="language tag implemented but not for all spans. https://datadoghq.atlassian.net/browse/AIT-8735",
+    )
```
Comment (robertomonteromiguel) [inline tests/test_semantic_conventions.py:234]: If you add only reason="AIT-8735" it would be fine

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]

File: tests/test_semantic_conventions.py (hunk 2 above)
Comment (daniel-romano-DD) [reply tests/test_semantic_conventions.py]: And yes, the errors came from this bug, not this PR related.

Comment (robertomonteromiguel) [reply tests/test_semantic_conventions.py]: I can see the error in dotnet here: FAILED tests/remote_config/test_remote_configuration.py::Test_RemoteConfigurationExtraServices::test_tracer_extra_services

could you correct the conflicts and launch the CI again?

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
I can see the error in dotnet here: FAILED tests/remote_config/test_remote_configuration.py::Test_RemoteConfigurationExtraServices::test_tracer_extra_services

could you correct the conflicts and launch the CI again?

**Review by robertomonteromiguel — APPROVED** [review]

**Pull request merged**